index.html

```diff
@@ -1761,11 +1741,31 @@
             "PresentationRequest" data-lt="start">select a presentation
             display</a>, the <a>user agent</a> MUST <dfn>monitor the list of
             available presentation displays</dfn> by running the following
-            steps.
+            steps:
           </p>
           <ol link-for="PresentationAvailability">
-            <li>Set the <a>list of available presentation displays</a> to the
-            empty list.
+            <li>Let <var>availabilitySet</var> be a shallow copy of the <a>set
+            of presentation availability objects</a>.
+            </li>
+            <li>If there is a pending request to <a for="PresentationRequest"
+            data-lt="start">select a presentation display</a> for a
+            <a>PresentationRequest</a> and if the <a>PresentationRequest</a>'s
+            <a>presentation display availability</a> is <code>null</code>, then
+            run the following substeps:
+              <ol>
+                <li>Let <var>A</var> be a newly created
```

Comment:
> Aggregating the monitoring steps across browsing contexts is a valid optimization, but do we need to specify it? Does that change any visible behavior?

It does not change visible behavior but it does change whether we need to aggregate multiple pending requests to `start()` in the spec.  Another way to put it: the spec should not be written to prohibit running the monitoring algorithm once to handle multiple pending `start()` requests.